<ui version="4.0" >
 <class>SelectCrystal</class>
 <widget class="QDialog" name="SelectCrystal" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Crystal Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QRadioButton" name="selectCrystalRadioButton" >
     <property name="text" >
      <string>Select a crystal:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="crystalListWidget" />
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="specifyParametersRadioButton" >
     <property name="text" >
      <string>Specify parameters:</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="1" >
      <widget class="QLineEdit" name="unitCellLineEdit" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Space group:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="spaceGroupLineEdit" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Unit cell:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2" >
     <item row="0" column="1" >
      <widget class="QLineEdit" name="valueLineEdit" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Value:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>buttonBox</zorder>
  <zorder>selectCrystalRadioButton</zorder>
  <zorder>specifyParametersRadioButton</zorder>
  <zorder>crystalListWidget</zorder>
  <zorder>line</zorder>
  <zorder>line_3</zorder>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>selectCrystalRadioButton</tabstop>
  <tabstop>crystalListWidget</tabstop>
  <tabstop>specifyParametersRadioButton</tabstop>
  <tabstop>unitCellLineEdit</tabstop>
  <tabstop>spaceGroupLineEdit</tabstop>
  <tabstop>valueLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelectCrystal</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectCrystal</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
